website/content/en/v0.35/concepts/nodepools.md

```diff
@@ -71,6 +71,11 @@ spec:
         - key: "karpenter.k8s.aws/instance-category"
           operator: In
           values: ["c", "m", "r"]
+          minValues: 2
+        - key: "karpenter.k8s.aws/instance-family"
+          operator: In
+          values: ["m5","m5d","c5","c5d","c4","r4"]
+          minValues: 5
```

Comment:
Worth giving a short, high-level comment above this line about what this does? That way someone doesn't have to go down and read the section if they don't want to to understand it

Reply:
Addressed.